h2o-docs/src/booklets/v2_2015/source/sw/sections/intro.tex

```diff
@@ -84,15 +85,15 @@ \subsection{Supported Data Formats}
   \item CSV
   \item SVMLight
   \item ARFF
+  \item Parquet
 \end{itemize}
 
 For more details, please refer to the H2O documentation at {\url{http://docs.h2o.ai}}.
 
 \subsection{Supported Spark Execution Environments}
 Sparkling Water can run on top of Spark in the following ways:
 \begin{itemize}
-  \item as a local cluster (where the master node is \texttt{local},
-\texttt{local[*]}, or \texttt{local-cluster[...]})
+  \item as a local cluster (where the master node is \texttt{local} or \texttt{local[*]})
```

Comment:
local-cluster is used just for testing, I think we should not expose it in the book